src/github.com/getlantern/flashlight/client/handler.go

```diff
@@ -11,18 +11,10 @@ import (
 
```

Comment:
All the changes to handler.go basically just revert the previous change to using direct fronting here.